Ft landing page

fab-ryan opened the pull request:
- [x] The landing page is implemented.
- [x] The landing page is responsive
- [x] The landing page contains the purpose of the app and other important links
- [x] Skeleton UIs/screen are/is shown when the page loads/is loading.

Commit:
setup react project add npm init

final configuration

not found page

Commit:
setup react project add npm init

final configuration

not found page

add dotenv

Creating yml file

For continuous integration and continuous deployment

starting landing page

add bach\gImage

custom button

icons packages

How does it work

added logout

new commit

factorization of folders

responsive nav bar

finish up responsiviness of page

Page not found add footer and navbar

added logout

home page/ page not found

Delete .github/workflows directory

Deleted github/workflows because CI/CD workflows not working so far.

setup redux in the project

added action increment/decrement on landing page

setup redux in the project

added action increment/decrement on landing page

ignored yarn.lock

Delete yarn.lock

add dotenv

Creating yml file

For continuous integration and continuous deployment

starting landing page

add bach\gImage

custom button

icons packages

How does it work

added logout

new commit

factorization of folders

responsive nav bar

finish up responsiviness of page

Page not found add footer and navbar

added logout

home page/ page not found

Delete .github/workflows directory

Deleted github/workflows because CI/CD workflows not working so far.

removed a bracket

setup react project add npm init

final configuration

not found page

setup react project add npm init

final configuration

not found pag

parent 7602cb8249ffca1ec6fdd8c8e670e5e29fa7e0c6
author fab-ryan <royalfabrice1234@gmail> 1649671239 +0200
committer fab-ryan <royalfabrice1234@gmail> 1650632323 +0200

setup react project add npm init

final configuration

not found page

add dotenv

Creating yml file

For continuous integration and continuous deployment

starting landing page

add bach\gImage

custom button

icons packages

How does it work

added logout

new commit

factorization of folders

responsive nav bar

finish up responsiviness of page

Page not found add footer and navbar

added logout

home page/ page not found

Delete .github/workflows directory

Deleted github/workflows because CI/CD workflows not working so far.

setup redux in the project

added action increment/decrement on landing page

setup redux in the project

added action increment/decrement on landing page

ignored yarn.lock

Delete yarn.lock

Creating yml file

For continuous integration and continuous deployment

starting landing page

add bach\gImage

custom button

icons packages

How does it work

added logout

new commit

factorization of folders

responsive nav bar

finish up responsiviness of page

added logout

home page/ page not found

Delete .github/workflows directory

Deleted github/workflows because CI/CD workflows not working so far.

removed a bracket

added logout

home page/ page not found

All finishup of skeleton ui and landing page

setup react project add npm init

final configuration

not found page

parent 7602cb8249ffca1ec6fdd8c8e670e5e29fa7e0c6
author fab-ryan <royalfabrice1234@gmail> 1649671239 +0200
committer fab-ryan <royalfabrice1234@gmail> 1650632323 +0200

setup react project add npm init

final configuration

not found page

add dotenv

Creating yml file

For continuous integration and continuous deployment

starting landing page

add bach\gImage

custom button

icons packages

How does it work

added logout

new commit

factorization of folders

responsive nav bar

finish up responsiviness of page

Page not found add footer and navbar

added logout

home page/ page not found

Delete .github/workflows directory

Deleted github/workflows because CI/CD workflows not working so far.

setup redux in the project

added action increment/decrement on landing page

setup redux in the project

added action increment/decrement on landing page

ignored yarn.lock

Delete yarn.lock

Creating yml file

For continuous integration and continuous deployment

starting landing page

add bach\gImage

custom button

icons packages

How does it work

added logout

new commit

factorization of folders

responsive nav bar

finish up responsiviness of page

added logout

home page/ page not found

Delete .github/workflows directory

Deleted github/workflows because CI/CD workflows not working so far.

removed a bracket

added logout

home page/ page not found

All finishup of skeleton ui and landing page

parent 7602cb8249ffca1ec6fdd8c8e670e5e29fa7e0c6
author fab-ryan <royalfabrice1234@gmail> 1649671239 +0200
committer fab-ryan <royalfabrice1234@gmail> 1650632323 +0200

setup react project add npm init

final configuration

not found page

add dotenv

Creating yml file

For continuous integration and continuous deployment

starting landing page

add bach\gImage

custom button

icons packages

How does it work

added logout

new commit

factorization of folders

responsive nav bar

finish up responsiviness of page

Page not found add footer and navbar

added logout

home page/ page not found

Delete .github/workflows directory

Deleted github/workflows because CI/CD workflows not working so far.

setup redux in the project

added action increment/decrement on landing page

setup redux in the project

added action increment/decrement on landing page

ignored yarn.lock

Delete yarn.lock

Creating yml file

For continuous integration and continuous deployment

starting landing page

add bach\gImage

custom button

icons packages

How does it work

added logout

new commit

factorization of folders

responsive nav bar

finish up responsiviness of page

added logout

home page/ page not found

Delete .github/workflows directory

Deleted github/workflows because CI/CD workflows not working so far.

removed a bracket

added logout

home page/ page not found

All finishup of skeleton ui and landing page

parent 7602cb8249ffca1ec6fdd8c8e670e5e29fa7e0c6
author fab-ryan <royalfabrice1234@gmail> 1649671239 +0200
committer fab-ryan <royalfabrice1234@gmail> 1650632323 +0200

setup react project add npm init

final configuration

not found page

add dotenv

Creating yml file

For continuous integration and continuous deployment

starting landing page

add bach\gImage

custom button

icons packages

How does it work

added logout

new commit

factorization of folders

responsive nav bar

finish up responsiviness of page

Page not found add footer and navbar

added logout

home page/ page not found

Delete .github/workflows directory

Deleted github/workflows because CI/CD workflows not working so far.

setup redux in the project

added action increment/decrement on landing page

setup redux in the project

added action increment/decrement on landing page

ignored yarn.lock

Delete yarn.lock

Creating yml file

For continuous integration and continuous deployment

starting landing page

add bach\gImage

custom button

icons packages

How does it work

added logout

new commit

factorization of folders

responsive nav bar

finish up responsiviness of page

added logout

home page/ page not found

Delete .github/workflows directory

Deleted github/workflows because CI/CD workflows not working so far.

removed a bracket

added logout

home page/ page not found

All finishup of skeleton ui and landing page

parent 7602cb8249ffca1ec6fdd8c8e670e5e29fa7e0c6
author fab-ryan <royalfabrice1234@gmail> 1649671239 +0200
committer fab-ryan <royalfabrice1234@gmail> 1650632323 +0200

setup react project add npm init

final configuration

not found page

add dotenv

Creating yml file

For continuous integration and continuous deployment

starting landing page

add bach\gImage

custom button

icons packages

How does it work

added logout

new commit

factorization of folders

responsive nav bar

finish up responsiviness of page

Page not found add footer and navbar

added logout

home page/ page not found

Delete .github/workflows directory

Deleted github/workflows because CI/CD workflows not working so far.

setup redux in the project

added action increment/decrement on landing page

setup redux in the project

added action increment/decrement on landing page

ignored yarn.lock

Delete yarn.lock

Creating yml file

For continuous integration and continuous deployment

starting landing page

add bach\gImage

custom button

icons packages

How does it work

added logout

new commit

factorization of folders

responsive nav bar

finish up responsiviness of page

added logout

home page/ page not found

Delete .github/workflows directory

Deleted github/workflows because CI/CD workflows not working so far.

removed a bracket

added logout

home page/ page not found

All finishup of skeleton ui and landing page

setup react project add npm init

final configuration

not found page

add dotenv

Creating yml file

For continuous integration and continuous deployment

starting landing page

add bach\gImage

custom button

icons packages

How does it work

added logout

new commit

factorization of folders

responsive nav bar

finish up responsiviness of page

Page not found add footer and navbar

added logout

home page/ page not found

Delete .github/workflows directory

Deleted github/workflows because CI/CD workflows not working so far.

setup redux in the project

added action increment/decrement on landing page

setup redux in the project

added action increment/decrement on landing page

setup react project add npm init

final configuration

not found page

parent 7602cb8249ffca1ec6fdd8c8e670e5e29fa7e0c6
author fab-ryan <royalfabrice1234@gmail> 1649671239 +0200
committer fab-ryan <royalfabrice1234@gmail> 1650632323 +0200

setup react project add npm init

final configuration

not found page

add dotenv

Creating yml file

For continuous integration and continuous deployment

starting landing page

add bach\gImage

custom button

icons packages

How does it work

added logout

new commit

factorization of folders

responsive nav bar

finish up responsiviness of page

Page not found add footer and navbar

added logout

home page/ page not found

Delete .github/workflows directory

Deleted github/workflows because CI/CD workflows not working so far.

setup redux in the project

added action increment/decrement on landing page

setup redux in the project

added action increment/decrement on landing page

ignored yarn.lock

Delete yarn.lock

Creating yml file

For continuous integration and continuous deployment

starting landing page

add bach\gImage

custom button

icons packages

How does it work

added logout

new commit

factorization of folders

responsive nav bar

finish up responsiviness of page

added logout

home page/ page not found

Delete .github/workflows directory

Deleted github/workflows because CI/CD workflows not working so far.

removed a bracket

added logout

home page/ page not found

All finishup of skeleton ui and landing page

parent 7602cb8249ffca1ec6fdd8c8e670e5e29fa7e0c6
author fab-ryan <royalfabrice1234@gmail> 1649671239 +0200
committer fab-ryan <royalfabrice1234@gmail> 1650632323 +0200

setup react project add npm init

final configuration

not found page

add dotenv

Creating yml file

For continuous integration and continuous deployment

starting landing page

add bach\gImage

custom button

icons packages

How does it work

added logout

new commit

factorization of folders

responsive nav bar

finish up responsiviness of page

Page not found add footer and navbar

added logout

home page/ page not found

Delete .github/workflows directory

Deleted github/workflows because CI/CD workflows not working so far.

setup redux in the project

added action increment/decrement on landing page

setup redux in the project

added action increment/decrement on landing page

ignored yarn.lock

Delete yarn.lock

Creating yml file

For continuous integration and continuous deployment

starting landing page

add bach\gImage

custom button

icons packages

How does it work

added logout

new commit

factorization of folders

responsive nav bar

finish up responsiviness of page

added logout

home page/ page not found

Delete .github/workflows directory

Deleted github/workflows because CI/CD workflows not working so far.

removed a bracket

added logout

home page/ page not found

All finishup of skeleton ui and landing page

parent 7602cb8249ffca1ec6fdd8c8e670e5e29fa7e0c6
author fab-ryan <royalfabrice1234@gmail> 1649671239 +0200
committer fab-ryan <royalfabrice1234@gmail> 1650632323 +0200

setup react project add npm init

final configuration

not found page

add dotenv

Creating yml file

For continuous integration and continuous deployment

starting landing page

add bach\gImage

custom button

icons packages

How does it work

added logout

new commit

factorization of folders

responsive nav bar

finish up responsiviness of page

Page not found add footer and navbar

added logout

home page/ page not found

Delete .github/workflows directory

Deleted github/workflows because CI/CD workflows not working so far.

setup redux in the project

added action increment/decrement on landing page

setup redux in the project

added action increment/decrement on landing page

ignored yarn.lock

Delete yarn.lock

Creating yml file

For continuous integration and continuous deployment

starting landing page

add bach\gImage

custom button

icons packages

How does it work

added logout

new commit

factorization of folders

responsive nav bar

finish up responsiviness of page

added logout

home page/ page not found

Delete .github/workflows directory

Deleted github/workflows because CI/CD workflows not working so far.

removed a bracket

added logout

home page/ page not found

All finishup of skeleton ui and landing page

parent 7602cb8249ffca1ec6fdd8c8e670e5e29fa7e0c6
author fab-ryan <royalfabrice1234@gmail> 1649671239 +0200
committer fab-ryan <royalfabrice1234@gmail> 1650632323 +0200

setup react project add npm init

final configuration

not found page

add dotenv

Creating yml file

For continuous integration and continuous deployment

starting landing page

add bach\gImage

custom button

icons packages

How does it work

added logout

new commit

factorization of folders

responsive nav bar

finish up responsiviness of page

Page not found add footer and navbar

added logout

home page/ page not found

Delete .github/workflows directory

Deleted github/workflows because CI/CD workflows not working so far.

setup redux in the project

added action increment/decrement on landing page

setup redux in the project

added action increment/decrement on landing page

ignored yarn.lock

Delete yarn.lock

Creating yml file

For continuous integration and continuous deployment

starting landing page

add bach\gImage

custom button

icons packages

How does it work

added logout

new commit

factorization of folders

responsive nav bar

finish up responsiviness of page

added logout

home page/ page not found

Delete .github/workflows directory

Deleted github/workflows because CI/CD workflows not working so far.

removed a bracket

added logout

home page/ page not found

All finishup of skeleton ui and landing page

setup react project add npm init

final configuration

not found page

add dotenv

Creating yml file

For continuous integration and continuous deployment

starting landing page

add bach\gImage

custom button

icons packages

How does it work

added logout

new commit

factorization of folders

responsive nav bar

finish up responsiviness of page

Page not found add footer and navbar

added logout

home page/ page not found

Delete .github/workflows directory

Deleted github/workflows because CI/CD workflows not working so far.

setup redux in the project

added action increment/decrement on landing page

setup redux in the project

added action increment/decrement on landing page

ignored yarn.lock

parent b529b143204518dce566d59a92ab5c846dfbb6a5
author UmuhireJessie <j.umuhire@alustudent.com> 1650622399 +0200
committer fab-ryan <royalfabrice1234@gmail> 1650635664 +0200

ignored yarn.lock

Delete yarn.lock

add dotenv

Creating yml file

For continuous integration and continuous deployment

add bach\gImage

custom button

icons packages

How does it work

added logout

new commit

factorization of folders

responsive nav bar

finish up responsiviness of page

added logout

home page/ page not found

Delete .github/workflows directory

Deleted github/workflows because CI/CD workflows not working so far.

removed a bracket

added logout

home page/ page not found

Delete .github/workflows directory

Deleted github/workflows because CI/CD workflows not working so far.

All finishup of skeleton ui and landing page

Anguandia commented on src/components/Headers/header.css:
```diff
@@ -83,6 +83,10 @@
   margin: 20px auto;
   padding: 10px 15px;
+  background-color: var(--light-gray);
+    margin-top: 70px;
```
The margin is too big please reduce

Anguandia reviewed (changes requested):
Also work on :
- Your PR description
- Squash your commits into 1

UmuhireJessie committed:
parent b529b143204518dce566d59a92ab5c846dfbb6a5
author UmuhireJessie <j.umuhire@alustudent.com> 1650622399 +0200
committer fab-ryan <royalfabrice1234@gmail> 1650635664 +0200

ignored yarn.lock

Delete yarn.lock

add dotenv

Creating yml file

For continuous integration and continuous deployment

add bach\gImage

custom button

icons packages

How does it work

added logout

new commit

factorization of folders

responsive nav bar

finish up responsiviness of page

added logout

home page/ page not found

Delete .github/workflows directory

Deleted github/workflows because CI/CD workflows not working so far.

removed a bracket

added logout

home page/ page not found

Delete .github/workflows directory

Deleted github/workflows because CI/CD workflows not working so far.

All finishup of skeleton ui and landing page

commit sqaush them

fab-ryan committed:
Merge branch 'develop' into ft-landing-page

fab-ryan committed:
Merge branch 'develop' into ft-landing-page
